readthedocs/templates/core/project_details.html

```diff
@@ -19,40 +19,42 @@
     <h3>{% trans "Versions" %}</h3>
   </div>
 
-  <div class="module-list">
+  <div class="module-list-wrapper">
+    <ul>
+    {% for version in versions|sort_version_aware %}
+      <li class="module-item col-span">
+      {% if version.uploaded or version.built %}
+        {# Link to the docs #}
+        <a class="module-item-title" href="{{ version.get_absolute_url }}">
+          {{ version.slug }}
+        {% if request.user in project.users.all %}
```

Comment:
You can use `{% if request.user|is_admin:project %}` for checking for the right user. You will need to load this tag ` {% load privacy_tags %}` (if itsn't already loaded)

Reply:
Yes. Right. I'll make the changes straight away.